doc/source/learning/openstack-helm.rst

```diff
@@ -0,0 +1,83 @@
+Openstack Helm
+=====================
+
+Openstack-helm is a collection of packages to deploy Opentack in kubernetes.
+
+These packages are written using the helm packaging structure called Chart that describe how to
+automatize the deployment the images.
+
+These images are created using LOCI
+
+
+List of interesting links:
+---------------------
+
+* Helm page
+    https://helm.sh
+
+* Helm documentation
+    https://helm.sh/docs/helm (v2)
+
+    https://v3.helm.sh/docs/ (v3)
+
+* Openstack helm page
+    https://wiki.openstack.org/wiki/Openstack-helm
+
+* Openstack helm infra repo
+    https://github.com/openstack/openstack-helm-infra
+
+* Openstack helm repo
+    https://github.com/openstack/openstack-helm
+
```

Comment:
probably worth adding -addons, -docs, and -images repos.

Reply:
I agree